pages/about-the-data.js

```diff
@@ -13,21 +14,89 @@ const About = () => (
     <Primary>
       <h1>{pageTitle}</h1>
       <p>
-        Increasing transparency in the criminal justice system is of vital importance to{' '}
-        <a href="http://texasjusticeinitiative.org/about/">TJI</a>, as our work relies on the use of information that
-        can be released to the public.
+        Increasing transparency in the criminal justice system is of vital importance to us at the Texas Justice
+        Initiative. Our work relies on the use of information that can be released to the public.
+      </p>{' '}
+      <p>
+        Currently on our website, <strong>we compile data about three main incident types that occur in Texas</strong>:
       </p>
+      <ol>
+        <li>
+          <strong>Injury of officer(s) and/or civilian(s): </strong>Any shooting of or by a Texas peace officer,
+          occurring while the officer is on- or off-duty, causing an injury.
+        </li>
+        <li>
+          <strong>Death of officer(s) and/or civilian(s): </strong> Any fatal shooting of or by a Texas peace officer,
+          occurring while the officer is on- or off-duty.
+        </li>
+        <li>
+          <strong>Death in custody: </strong>
+          Any death of an individual while they are in a penal institution, in the custody of a peace officer or as a
+          result of a peace officer’s use of force, i n a jail, correctional facility, or state juvenile facility dies.
+        </li>
+      </ol>
       <p>
-        State laws allow anyone to get a good glimpse at the criminal justice system, though plenty of hurdles remain.
-        Through open records requests, TJI has obtained data from various governmental agencies. We then clean, analyze
-        and present the data – both in the form of a full data set and in comparative graphics – to the public. Here is
-        a description of our current data sets:
+        After an incident, the governmental agency in which it occurred (i.e. a county jail etc.) is required to file a
+        report for the respective incident (
+        <a
+          href="https://drive.google.com/file/d/1qRRN6HJpVTXvo4G5MzJGLwAHSPEHvNNR/view"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <strong>1</strong>
+        </a>
+        ,{' '}
+        <a
+          href="https://drive.google.com/open?id=1iBJQ6zl1UQOEUnd18Krb9PnZEL8Nd0kY"
```

Comment:
For this link and the next one, I'm prompted to log in to Google. Do we need to update permissions here?

Reply:
@EvaRuthM can you take a look at this for us?

Reply:
just changed this to public